pipeline/analyze_assembly.py

```diff
@@ -118,24 +126,34 @@ def load_overlaps(self, filename,  min_ident,  comparison =best_hit, min_length=
 		     self.overlaps_identity[name2] = ident
        		     self.contigs_overlaps[name2] +=1  
 
-        for name2 in self.overlaps_s1  :
-           name1 = self.overlaps_genome[name2]
+        # iterate over all the overlaps,
+        for name2 in self.overlaps_s1:
+
+            # get the best (highest identity) overlap
+            name1 = self.overlaps_genome[name2]
             
-           if (name1 in covered) and (name2 in aligned) and (name1 != "null") and comparison( self.contigs_overlaps[name2],1) :
+            # @CTB when would name1/name2 not be in covered/aligned?
+            # @CTB note, I don't understand how comparison is used here
```

Comment:
no_overlap is the no mis-assemblies, the comparison function tests if we have only one alignment only, if no-overlap is called. 
besthit is the default, comparison returns true if he we have more one or more alignment, but the best one will be selected.